### DISCUSSION TITLE
Deterioration in connection on many servers?

### DISCUSSION THREAD

**Discussion by JacquesFreud** [jamulussoftware/jamulus#3209] [Technical Support & Help] [answered]
I am writing this on behalf of a rather widespread group of West-European users. We seem to notice a deterioration in quality (more jitter, 'blubbering' etc.) on servers that always had been reliable. Most are located in Germany. Is this just a coincidence or has this problem been noticed by other people too or even the Jamulus developers team?

**Comment by mcfnord** [comment]
Sorry to hear of these problems. Another German experienced this today on a server in Köln. There are probably other (non-Jamulus) services impacted by this ongoing network disruption.

**Comment by ann0see** [comment]
I don't believe we can do much on the software side. But can you check where the servers in question are hosted? Maybe the hosting provider/some ISPs have degraded performance.

**Reply by JacquesFreud** [reply]
Yes, could be. But the problem seems to concern quite a few servers, hence my question. Must be the Russians.

**Reply by rdica** [reply]
Just curious, are the affected servers all coming from the same hosting service?

**Reply by JacquesFreud** [reply]
No idea...how does one check this?

**Reply by rdica** [reply]
You can get the IP address of each server from https://explorer.jamulus.io, run a whois on each and see who owns the IP. If you're unfamiliar with the whois lookup (can be done on the web or on the command line if you have those tools), list the affected servers here and I can look them up.

**Reply by JacquesFreud** [reply]
Thanks. I saw a few of the same IP-addresses of a couple of the affected servers. That is of servers I think that are affected since we are just noticing the problem these last few weeks. I have no list, really and I would not want you to spend time on something that is just a suspicion. We noticed problems on servers like MTKS.F, St. Aposteln and Jamulix. They all used to be good and started making trouble more or less at the same time. But as our attention was always more fixed on playing, I don't think anybody took the trouble of writing down which servers we are talking about...

**Reply by rdica** [reply, marked as answer]
The Jamulix and St. Aposteln servers are all run from the same host, apparently hosted by Uvensys, MTKS.F is run on an Amazon AWS instance. The only commonality I can find is both are hosted in Germany.

I can speculate about the Uvensys-hosted servers; the host running them may be overwhelmed (perhaps needs more resources like CPU or mem), or the network the hoster uses is saturated, or their peering/interconnects with upstream networks possibly saturated.

Not very helpful, I know, but maybe gives you an overview of things that can impact Jamulus use that are out of the control of users. Being a network-centric application, we are all subject to the whims of networks peering/routing arrangements (these are based on costs mostly and not the most efficient route, sadly).

**Reply by JacquesFreud** [reply]
That is very kind of you, looking up the hosting details of the servers. I think we will have to deal with the limitations.
